content/_help/manage-your-account/personal-key._es.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: help
+title: Clave personal 
+category: trouble-signing-in
+permalink: /es/help/manage-your-account/personal-key/
+order: 3 
+redirect_from:
+  - /es/es/help/manage-your-account/personal-key/
+---
+Una clave personal es un código de 16 caracteres que encripta tu información personal con tu cuenta. Tu clave personal es la única forma de acceder a tu información si pierdes u olvidas tu contraseña.
+
+Ten en cuenta que:
+
+- Tendrás que verificar tu identidad nuevamente si restableces tu contraseña sin tu clave personal.
+- Las claves personales solo se pueden usar una vez. Si anteriormente utilizaste una clave personal para restablecer tu contraseña, ya tienes una nueva clave personal.
+- Puedes revisar tu clave personal actual en la página de tu cuenta.
+
+Recibes tu clave personal luego de verificar tu identidad. Guarda y mantén tu clave personal en un lugar seguro. Puedes copiarla, descargarla o imprimirla.
+
+## Cambiar o restablecer tu contraseña
+
+### Si tienes una clave personal
+
+Cuando restablezcas tu contraseña, necesitarás tu clave personal para acceder a tu información.
+
+1. Después de que hayas restablecido tu contraseña, inicia sesión con tu nueva contraseña.
+2. En la página de tu cuenta, te aparecerá un aviso de que tu cuenta se desactivó recientemente debido a que se restableció la contraseña.
+3. Selecciona "Reactivar tu perfil ahora".
+4. Selecciona "Tengo mi clave"
+5. Ingresa tu clave personal más reciente. Si restableciste tu contraseña anteriormente, es posible que tengas una nueva clave personal.
+6. Vuelve a ingresar tu contraseña.
+7. Asegúrate de guardar tu nueva clave personal después de restablecer tu contraseña.
+
+### Si no tienes una clave personal
+
+Si restableces tu contraseña y no tienes tu clave personal, tendrás que verificar nuevamente tu identidad.
+
+1. Después de que hayas restablecido tu contraseña, inicia sesión con tu nueva contraseña.
+2. En la página de tu cuenta, te aparecerá un aviso de que tu cuenta se desactivó recientemente debido a que se restableció la contraseña.
+3. Selecciona "Reactivar tu perfil ahora".
+4. Selecciona "No tengo mi clave".
+5. Selecciona "Continuar" para comenzar de nuevo el proceso de verificación de identidad.
+
+## Related articles
+
+[Forgot your password](/es/help/trouble-signing-in/forgot-your-password/)
+[Change your password](/es/help/manage-your-account/change-your-password/)
+[How to verify your identity](/es/help/verify-your-identity/how-to-verify-your-identity/)
```

Comment:
Should these link texts be in Spanish?

Reply:
This is how the link shows up for me when I switch to the languages at the top dropdown, so I think the links are valid as they are. Spanish example: https://login.gov/es/help/trouble-signing-in/forgot-your-password/